### ISSUE TITLE
[BUG/ISSUE] Nested-grid simulation without boundary condition

### ISSUE THREAD

**Issue by jiaying002** [geoschem/geos-chem#1671]
I use GEOSChem v.14.0.0 to run a nested-grid simulation on a custom lat and lon at 0.5*0.625 resolution under GISS E2.1 mode.

From this website(http://wiki.seas.harvard.edu/geos-chem/index.php/Setting_up_GEOS-Chem_nested_grid_simulations), it seems like all the simulations using 0.5*0.625 resolution should use boundary conditions. However, I found out that my simulation could run perfectly without using the boundary conditions. 

From the GC.log, my simulation did not read the bc files. Also, I deleted the path to my boundary conditions folder in the HEMCO_config.rc to test if it uses the bc files, the result is my simulation runs continuously and could generate readable output files. I attached the files that may be crucial. I'd like to know if there is a bug here or if there is something I don't know.
![image](https://user-images.githubusercontent.com/113654537/221421384-233e1593-e504-4518-8f17-113e8f2ba05b.png)
（SSP119-20400205 SO4 map from the newest output file run without boundary condition）

Thanks in advance.

[GC.log](https://github.com/geoschem/geos-chem/files/10833827/GC.log)
[HEMCO_Config.rc.txt](https://github.com/geoschem/geos-chem/files/10833833/HEMCO_Config.rc.txt)
[geoschem_config.yml.txt](https://github.com/geoschem/geos-chem/files/10833835/geoschem_config.yml.txt)
[HISTORY.rc.txt](https://github.com/geoschem/geos-chem/files/10833838/HISTORY.rc.txt)
[HEMCO_Config.rc.gcap2_metfields.txt](https://github.com/geoschem/geos-chem/files/10833839/HEMCO_Config.rc.gcap2_metfields.txt)

**Comment by yantosca** [comment]
Thanks for writing @jiaying002.  While it is possible that your simulation may run OK without boundary conditions, this is not recommended.  The transport scheme TPCORE will attempt to move mass from beyond the edge of your nested grid domain.  Without boundary conditions it is probably copying junk, thus making your simulation useless. 

NOTE: We are now asking users to identify themselves and their institutions when requesting help. Please do this in the future. Thanks!

**Comment by jiaying002** [comment]
Thank you so much for your reply!

I am Jiaying Li, currently a first-year Ph.D. student, from Nanyang Technological University. I tried again after your suggestion, and I got those warnings. I'd like to know if it is normal to have those warnings for boundary condition. Did I successfully use the boundary conditions?

![image](https://user-images.githubusercontent.com/113654537/221631189-7f15c810-1145-45f7-97c5-5d1e429e5c26.png)
